api/src/devcomp/application/passages/index.js

```diff
@@ -26,6 +28,32 @@ const register = async function (server) {
         tags: ['api', 'passages', 'modules'],
       },
     },
+    {
+      method: 'POST',
+      path: '/api/passages/{passageId}/answers',
+      config: {
+        auth: false,
+        validate: {
+          params: Joi.object({
+            passageId: identifiersType.passageId.required(),
+          }).required(),
+          payload: Joi.object({
+            data: Joi.object({
+              attributes: Joi.object({
+                'element-id': Joi.string().required(),
```

Comment:
Je sais que c'est dans un fichier actuellement et non pas côté database, mais j'aurais envie quand même de lui indiquer que le type est de type `typesAlphanumeric255` (identifiersType). Sans ça à la lecture, "ma phrase très longue <script>toto</script>" passerait ici la validation Joi.

Reply:
Pour être encore plus précis, on va partir sur :

```suggestion
                'element-id': Joi.string().guid().required(),
```

Reply:
> Pour être encore plus précis, on va partir sur :

Même : 
```suggestion
                'element-id': Joi.string().guid({ version: 'uuidv4' }).required(),
```